src/views/Public/Index.vue

```diff
@@ -250,16 +250,22 @@ export default {
     },
 
     createNewChart () {
-      const { namespaceID } = this.namespace
-      const newChart = new compose.Chart({
-        namespaceID,
-        name: 'Demo Chart',
-        handle: 'demo_chart',
-      })
+      if (this.hasModules) {
```

Comment:
Since step is disabled if `!hasModules`. No need for this. Or does an error occur if this if is not present ?

Reply:
No error occurres, removing if statement